src/scripts/http/cachedHttp.ts

```diff
@@ -104,6 +106,19 @@ export class CachedHttp {
 			return undefined;
 		}
 
+		const userAuthType: string = valueAsJson.authType;
```

Comment:
So basically this piece of code will only be called when there is a need for a fresh value right? code here https://github.com/OneNoteDev/WebClipper/blob/dc7c9feafcbf4168712165f210afa43ae6908a8d/src/scripts/http/cachedHttp.ts#L28, which means there can still be no userinfo till the time cache is not expired. Is it fine or should we add the upn during boot also?

Reply:
I see that whenever we retrieve user info with a new access token then it is necessary to decode the token and add that info to userinfo. For the cached token it does not make much sense to do that and eventually freshtoken will be called for the cached one as well once expired. So it is not necessary to handle for that.
Let me know if you feel otherwise